lms/djangoapps/discussion_api/api.py

```diff
@@ -415,7 +415,7 @@ def get_comment_list(request, thread_id, endorsed, page, page_size, mark_as_read
         raise Http404
     num_pages = (resp_total + page_size - 1) / page_size if resp_total else 1
 
-    results = [CommentSerializer(response, context=context).data for response in responses]
+    results = [CommentSerializer(response, remove_fields=["children"], context=context).data for response in responses]
```

Comment:
A goal of this PR was to improve the performance of retrieving responses with many comments. If we can't exclude the comments then trying to load a response with many comments such as an "introduce yourself" thread will take a long time. If we can't retrieve responses without the comments, then maybe we will have to update the comments service.